pixie/targets/bsd_utils.py

```diff
@@ -0,0 +1,32 @@
+from ctypes import CDLL, c_char_p, c_void_p, create_string_buffer, byref, c_uint
+from ctypes.util import find_library
+
+libc = CDLL(find_library("c"))
+
+# Define the sysctlbyname function signature
+libc.sysctlbyname.argtypes = [
+    c_char_p,  # name
+    c_void_p,  # oldp
+    c_void_p,  # oldlenp
```

Comment:
This should be `size_t *`?

Reply:
fixed in https://github.com/numba/pixie/pull/28/commits/4b276540796a1aca60c2e95821695fc35cb607a0